core-common/src/test/java/org/apache/kylin/common/util/HiveCmdBuilderTest.java

```diff
@@ -59,14 +59,14 @@ public void testHiveCLI() {
         hiveProps.put("hive.execution.engine", "mr");
         Map<String, String> hivePropsOverwrite = new HashMap<>();
         hivePropsOverwrite.put("hive.execution.engine", "tez");
-        HiveCmdBuilder hiveCmdBuilder = new HiveCmdBuilder();
+        HiveCmdBuilder hiveCmdBuilder = new HiveCmdBuilder("test HiveCLI");
         hiveCmdBuilder.addStatement("USE default;");
         hiveCmdBuilder.addStatement("DROP TABLE `test`;");
         hiveCmdBuilder.addStatement("SHOW\n TABLES;");
         hiveCmdBuilder.setHiveConfProps(hiveProps);
         hiveCmdBuilder.overwriteHiveProps(hivePropsOverwrite);
         assertEquals(
-                "hive -e \"USE default;\nDROP TABLE \\`test\\`;\nSHOW\n TABLES;\n\" --hiveconf hive.execution.engine=tez",
+                "hive -e \"set set mapred.job.name='test HiveCLI';\nUSE default;\nDROP TABLE \\`test\\`;\nSHOW\n TABLES;\n\" --hiveconf hive.execution.engine=tez",
```

Comment:
why "set" twice?

Reply:
typo. I will fix it in the updated PR.

Reply:
It is a typo. I will fix it in the updated PR.